### ISSUE TITLE
\kvtcb@doc@description is not init to empty per {docXXX} env

### ISSUE THREAD

**Issue by muzimuzhi** [T-F-S/tcolorbox#90]
In the documentation for option "doc label", page 487 of `tcolorbox.pdf`, v4.22, the `\kvtcb@doc@description` set by outer `{docTcbKey}` environment affects inner `{docPathOperation*}` environment.
![image](https://user-images.githubusercontent.com/6376638/71785336-62f95d00-3039-11ea-96ed-bf2523555a67.png)

Since `{docXXX}` environments may nest, options "doc description" should be initialized per environment.

------
An extracted example
```tex
\documentclass{article}
\usepackage{tcolorbox}
\tcbuselibrary{documentation}

\begin{document}
% copied from tcolorbox.doc.s_main
\tcbmakedocSubKey{docTcbKey}{tcb}

% copied from tcolorbox.doc.documentation.tex
\begin{docTcbKey}[][doc new={2019-09-18}]{doc label}{=\meta{text}}{no default, initially unset}
  If used inside the option list of \refEnv{docCommand}, \refEnv{docEnvironment},
  \refEnv{docKey}, or \refEnv{docPathOperation}, then \meta{text} is used
  for labeling instead of the name of the definition.
\begin{dispExample}
\begin{docPathOperation*}[doc label=pathline]{-{}-}{\meta{coordinate or cycle}}
  This is the documentation of \refPathOperation{pathline}.
\end{docPathOperation*}
\end{dispExample}
\end{docTcbKey}
\end{document}
```

**Comment by T-F-S** [comment]
Yes, you are right. I will correct this for the next version of `tcolorbox`.

Thank you for reporting.

**Comment by T-F-S** [comment]
Resolved for version 4.30 (2020-04-28)
